src/bin/substudy.rs

```diff
@@ -21,7 +21,7 @@ Subtitle processing tools for students of foreign languages
 
 Usage: substudy clean <subs>
        substudy combine <foreign-subs> <native-subs>
-       substudy export csv <video> <foreign-subs> [<native-subs>]
+       substudy export csv [--limit=<secs>] <video> <foreign-subs> [<native-subs>]
        substudy export review <video> <foreign-subs> [<native-subs>]
        substudy export tracks <video> <foreign-subs>
```

Comment:
Two thoughts:

- Can we find a better name than `--limit`?
- If we're going to support this for `export csv`, can we support it for all the `export` subcommands? I think they mostly use the same code internally, don't they?